data/books.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "publish_date": "2022-12-12",
+    "genre": "Horror",
+    "label_title": "Book1",
+    "label_color": "Red",
+    "author_first_name": "Jack",
+    "author_last_name": "Mile",
+    "publisher": "pub1",
+    "cover_state": "good"
+  }
+]
```

Comment:
- Great Job for being able to display the JSON data, I think you forget to save what you added, so please save it to a JSON file to be able to get what we added after quitting the app 👍

Reply:
Hi @binhussen, Sorry, I don't get it, the app saves the data the moment you quit the app if that's what you meant. It would be nice if you could help us with more info. Thanks!.
![image](https://github.com/Javed-Shahbaz/Ruby-Group-Capstone/assets/25826025/824b4c10-9722-4fcc-840c-b140bf684b10)